Sources/SwiftKuery/Table.swift

```diff
@@ -159,19 +294,53 @@ open class Table: Buildable {
         return self
     }
     
-    /// Add a primary key to the table.
-    ///
-    /// - Parameter columns: Columns that constitute the primary key.
-    /// - Returns: A new instance of `Table`.
+    /**
+     Function to set a single `Column` instances` as a primary key, in the `Table` instance.
```

Comment:
There is an extraneous ` after the word instances which is messing up the formatting.